static/js/twitter_filter.js

```diff
@@ -0,0 +1,24 @@
+function twitter_filter() {
+  var twitter_handles = [];
+  for(var i=0; i<document.getElementsById("org-info"); i++) {
+    var org_info = document.getElementsById("org-info");
+    for(var j=0; i<org_info.children.length; i++) {
+      if(org_info.children[j].href.startsWith("https://twitter.com/")) {
+        twitter_handles.push(org_info.children[j].href.substr(20));
+      }
+    }
+  }
+  console.log(twitter_handles);

```

Comment:
Shouldn't `console.log` in front-end JS

Reply:
@andrewda went afk, still isn't finished yet. Have yet to remove debug lines